test/plugin/scenarios/webflux-scenario/config/expectedData.yaml

```diff
@@ -14,30 +14,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 registryItems:
   applications:
-    - {webflux-projectB-scenario: nq 0}
-    - {webflux-projectA-scenario: nq 0}
+    - {webflux-scenario: nq 0}
   instances:
-    - {webflux-projectB-scenario: 1}
-    - {webflux-projectA-scenario: 1}
+    - {webflux-scenario: nq 0}
   operationNames:
-    - webflux-projectB-scenario: [org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.success,
-                                  org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.hello,
-                                  WEBFLUX.handle, org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.error,
-                                  org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.healthCheck,
-                                  RoutingConfiguration.org.apache.skywalking.apm.testcase.sc.webflux.projectB.route.TestHandler]
-    - webflux-projectA-scenario: [/testcase/annotation/mono/hello, /projectA/testcase,
-                                  /testcase/annotation/success, /testcase/annotation/healthCheck, /notFound, /projectA/healthCheck,
-                                  /testcase/route/error, /testcase/route/success, /testcase/annotation/error]
+    - webflux-scenario: ['GET:/testcase/route/bad', 'GET:/testcase/annotation/bad',
+                         'HEAD:/healthCheck', 'GET:/testcase/annotation/hello', 'GET:/testcase/notFound',
+                         'GET:/testcase', 'GET:/testcase/route/hello']
   heartbeat: []
 segmentItems:
-  - applicationCode: webflux-projectB-scenario
+  - applicationCode: webflux-scenario
     segmentSize: nq 0
     segments:
       - segmentId: not null
         spans:
-          - operationName: org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.success
+          - operationName: HEAD:/healthCheck
```

Comment:
The healthCheck should not add to the expected data, I know it was there, but it actually should not. Because, we can’t guarantee the agent will collect the health check request, actually. The key of healthCheck is making sure the container is ready to test. Make sense?

Reply:
Ok, I will delete healthCheck test case